diff -Nur SDL_Pango-0.1.2.orig/src/Makefile.am SDL_Pango-0.1.2.am/src/Makefile.am
--- SDL_Pango-0.1.2.orig/src/Makefile.am	2004-12-10 11:06:33.000000000 +0200
+++ SDL_Pango-0.1.2.am/src/Makefile.am	2006-10-29 16:56:01.000000000 +0200
@@ -2,7 +2,7 @@
 
 lib_LTLIBRARIES = libSDL_Pango.la
 
-libSDL_Pangoincludedir = $(includedir)
+libSDL_Pangoincludedir = $(includedir)/SDL
 libSDL_Pangoinclude_HEADERS = SDL_Pango.h
 
 libSDL_Pango_la_SOURCES = SDL_Pango.c
